package.json

```diff
@@ -6,6 +6,7 @@
     "bin": "bin.js",
     "scripts": {
         "test": "jest",
```

Comment:
Think we can delete the old testing flow now, right? (the `bin.test.ts` file, `.travis.yml`, and this script)
Then we can call `test:e2e` simply `test`.

Reply:
At the time I made this PR I was so relieved that everything was working that I forgot some of the original goals I'd had.

I think there are still some tests from `bin.test.ts` that I haven't replicated in this new system yet. Before we merge this PR I will achieve feature parity, then we can delete `bin.test.ts`.

As for renaming `test:e2e` simply `test`, I'd rather avoid that. Eventually I'd like to implement unit tests as well, `test:unit`. And then have `test` as an alias to run both unit and e2e tests.